spotty/providers/aws/cfn_templates/instance/template.py

```diff
@@ -56,20 +56,16 @@ def prepare_instance_template(ec2, instance_config: InstanceConfig, docker_comma
             }]
         del template['Resources']['InstanceLaunchTemplate']['Properties']['LaunchTemplateData']['SecurityGroupIds']
 
-    # add ports to the security group
-    for port in instance_config.ports:
-        if port != 22:
-            template['Resources']['InstanceSecurityGroup']['Properties']['SecurityGroupIngress'] += [{
-                'CidrIp': '0.0.0.0/0',
-                'IpProtocol': 'tcp',
-                'FromPort': port,
-                'ToPort': port,
-            }, {
-                'CidrIpv6': '::/0',
-                'IpProtocol': 'tcp',
-                'FromPort': port,
-                'ToPort': port,
```

Comment:
Just noticed that now if the `inboundIp` parameter is not specified, we're not opening ports for IPv6. So, I think we need to use `None` for the default value instead of `0.0.0.0/0` to know when the parameter wasn't specified and open ports for both protocols.